gotham/src/lib.rs

```diff
@@ -1,6 +1,6 @@
-//! Gotham - A flexible web framework that does not sacrifice safety, security or speed.
+//! Gotham &ndash; A flexible web framework that does not sacrifice safety, security or speed.
 //!
-//! You can find out more about Gotham, including where to get help,  at https://gotham.rs.
+//! You can find out more about Gotham, including where to get help, at <https://gotham.rs>.
```

Comment:
Phrase itself has since changed to:

"A flexible web framework that promotes stability, safety, security and speed."